test_logwarn

```diff
@@ -34,5 +34,5 @@ echo $out | grep -q '\[21257:warn\] Error on AMQP channel received: Failed closi
 echo $out | grep -q '\[31798:warn\] no products found, retrying version wildcard' && exit 2
 echo $out | grep -q '\[14008:warn\] START_AFTER_TEST=RAID0:ppc64le not found - check for typos and dependency cycles' && exit 2
 echo $out | grep -q 'Worker not found for given connection during connection close' && exit 2
-
+echo $out | grep -q '\[15550:warn] AMQP connection error: SSL connect attempt failed with unknown error' && exit 2
```

Comment:
To make the test useful you should include an example line in the file `in.log`. It's obvious that this message is not shown when it is not in the original log file.